about.html

```diff
@@ -0,0 +1,218 @@
+        <p>Because DeveloperWeek covers all new technologies, our conference and workshops invite you to get intro
+          lessons (or advanced tips and tricks) on technologies like VR Dev, Artificial Intelligence Dev, Blockchain
+          Dev, IoT Dev, Serverless technology, Microservices technology, new JavaScript frameworks, and more.</p>
+        <p>This year, DeveloperWeek will have an expanded DevExec World, which is 2 days of roundtable talks,
+          educational talks, and networking events for engineering managers, tech executives, and lead developers.
+          Additionally, DeveloperWeek will be co-located with ProductWorld and CloudWorld.</p>
+        <p>DeveloperWeek puts the spotlight on new technologies. Companies that participated in past DeveloperWeek
+          events include Google, Facebook, Yelp, Rackspace, IBM, Cloudera, Red Hat, Optimizely, SendGrid, Blackberry,
+          Microsoft, Neo Technology, Eventbrite, Klout, Built.io, Ripple, GNIP, Tagged, HackReactor, and 30+ more here!
+        </p>
+        <p>When it comes to technology, there’s incremental change, and then there’s fundamental innovation. Developer
+          technology, from blockchain and artificial intelligence to big data and quantum computing represents
+          fundamental innovation that people can build on for years. We are in the DevTech Age, where developer
+          technologies and tools are now the most disruptive and fundamental technology innovation in the marketplace.
+          When you build tools for developers, you are not just implementing a small incremental use case, you are
+          building platforms, frameworks, and APIs that will enable entirely new web, mobile, and IoT innovation.</p>
+      </div>
+      <div class="location-about">
+        <div class="dateinfo">Contact the DeveloperWeek 2023 Team</div>
+        <div class="locate">Contact us</div>
+      </div>
+    </section>
+  </div>
+
+  <div class="wrap past-winners">
+    <section>
+      <h2 class="normal">Past DEVIES Winners</h2>
+      <div class="guide-bar"></div>
+      <div class="past-winners-list">
+        <div class="past-winner">
+          <a href="#">
+            <button>2022 Devies Winners</button>
+          </a>
+        </div>
+        <div class="past-winner">
+          <a href="#">
+            <button>2021 Devies Winners</button>
+          </a>
+        </div>
+        <div class="past-winner">
+          <a href="#">
+            <button>2020 Devies Winners</button>
+          </a>
+        </div>
+        <div class="past-winner">
+          <a href="#">
+            <button>2019 Devies Winners</button>
+          </a>
+        </div>
+        <div class="past-winner">
+          <a href="#">
+            <button>2018 Devies Winners</button>
+          </a>
+        </div>
+      </div>
+    </section>
+  </div>
+
+  <footer class="site-footer">
+    <div class="wrap footer-black">
+      <div class="flex column">
+
```

Comment:
I like your design! but don't you think you can improve it by following the Behance design? In other words, you can update the layout and size of the footer.

![image](https://user-images.githubusercontent.com/34779928/197263180-fef96877-543a-45ff-a588-57c0f82fa847.png)